### DISCUSSION TITLE
After reload of filament into runout sensor no resume option.

### DISCUSSION THREAD

**Discussion by steve-eaton** [Jyers/Marlin#991] [General Discussion]
Once the filament is reloaded Marlin will refresh and loose all printing data. This will make it so there is no resume option.

**Comment by Cal-LF** [comment]
Do you have the latest version installed?

**Comment by steve-eaton** [comment]
Thank you, I just updated from 1.3.3 to 1.3.4 and that seemed to have fixed
the issue.
